src/components/Legend/LegendContainer.tsx

```diff
@@ -22,5 +33,48 @@ export default function LegendContainer({ items, loading = false }: LegendContai
         }))}
       />
     </div>
+  ) : (
+    <>
+      <div className="absolute z-50 bottom-3 right-16">
+        <Tooltip text={INFO_POPUP_TOOLTIP}>
+          <Button
+            onClick={() => setInfoPopup(true)}
+            className="
+        relative flex items-center justify-center min-w-12 h-12 px-1 rounded-full bg-content1 shadow-md"
+          >
+            <Information size={32} />
```

Comment:
not sure about this icon, it indicates something urgent to me. Maybe we can use `InfoIcon.svg` here? @marinovl7 what do you think?

Reply:
Since in our mechanism we only know it is in mobile version depends on the resolution, when screen width smaller than 640 px e.g. If the use open the website in small screen in laptop, then we need show tooltip although is very unusual.

Reply:
See my comment in #56. Lets wait on that solution!